libs/lib-client/diagram/src/lib/diagram/diagram-row.component.html

```diff
@@ -32,34 +32,44 @@
     [ngForOf]="groupedTiles?.dropZone"
     [ngForTrackBy]="trackTileBy"
   >
-    <flogo-diagram-tile-task
-      cdkDrag
-      cdkDragBoundary="flogo-diagram"
-      @tileTask
-      [class.is-disabled]="isDragging && disableTile(tile?.task?.id)"
-      [tile]="tile"
-      [cdkDragData]="tile?.task?.id"
-      [currentSelection]="selection"
-      [isReadOnly]="isReadOnly"
-      [icon]="iconProvider?.getIconUrlById(tile?.task?.id)"
-      (select)="onTaskSelected($event)"
-      (branch)="onTaskAction($event)"
-      (remove)="onTaskAction($event)"
-      (configure)="onTaskAction($event)"
-      (cdkDragStarted)="onTaskDragStart($event)"
-    >
-      <flogo-diagram-tile-placeholder
-        *cdkDragPlaceholder
-      ></flogo-diagram-tile-placeholder>
-      <ng-template cdkDragPreview [matchSize]="true">
-        <flogo-diagram-tile-preview
-          [id]="tile?.task?.id"
-          [title]="tile?.task?.title"
-          [icon]="iconProvider?.getIconUrlById(tile?.task?.id)"
-          [isSubflow]="tile?.task?.features?.subflow"
-        ></flogo-diagram-tile-preview>
-      </ng-template>
-    </flogo-diagram-tile-task>
+    <div class="tile--group">
```

Comment:
you could've probably avoid this class by adding a class in the div at line 20

Reply:
this new div element is needed to absolutely position the add between button close to the tile (half of the button is on top of the tile)

![image](https://user-images.githubusercontent.com/47237691/86704768-d739ab00-c032-11ea-80aa-d511e1a37b2b.png)